app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        self.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f""
+                f"{{Name: {self.name},"
+                f" Health: {self.health},"
+                f" Hidden: {self.hidden}}}")
+
+
+class Herbivore(Animal):
+
+    def hide(self) -> None:
+        self.hidden = not self.hidden
+
+
+class Carnivore(Animal):
+
+    def bite(self, animal: Animal) -> None:
+        if isinstance(animal, Carnivore):
```

Comment:
Rewrite your code without return 
<img width="315" alt="Screenshot 2023-10-11 at 17 07 07" src="https://github.com/mate-academy/py-herbivores-and-carnivores/assets/117740672/8edc49cb-9606-4909-bc0f-64ff2f843f92">